pytorch_lightning/strategies/strategy_registry.py

```diff
@@ -130,8 +130,26 @@ def is_register_strategies_overridden(strategy: type) -> bool:
     return strategy_attr.__code__ is not super_attr.__code__
 
 
+@classmethod
+def register_strategies(cls, strategy_registry: Dict) -> None:
+    strategy_registry.register(
+        cls.strategy_name,
```

Comment:
strategy name is currently not required on the API. This will break if users don't have a `strategy_name` defined. It also doesn't make sense to require `strategy_name` since strategies may have multiple nicknames they want to register.

Reply:
> doesn't make sense to require strategy_name since strategies may have multiple nicknames they want to register.

Users could always implement the `register_strategies` method to add more aliases for Strategy classes.